code/dataset.py

```diff
@@ -50,14 +48,19 @@ def load_dataloaders(data_dir,
                      batch_size=1024,
                      split_trainvalid=0.90,
                      t_testsplit = 5,
-                     sample_uniform_action=False):
+                     sample_uniform_slate=False):
 
     logging.info('Load data..')
-    data = torch.load(f'{data_dir}/data.pt')
-    dataset = SequentialDataset(data, sample_uniform_action)
+    with np.load(f'{data_dir}/data.npz') as data_np:
```

Comment:
If we want to support older Python versions than `3.7`, we should probably write this as `'{}/data.npz'.format(data_dir)`

Reply:
bah agree, but its so much nicer with f"". should be mandatory with 3.7+